### DISCUSSION TITLE
Displaying location field values in Modern Search

### DISCUSSION THREAD

**Discussion by patrikhellgren** [microsoft-search/pnp-modern-search#2491] [Q&A] [answered]
I know this is not really a Modern Search question but hopefully someone has some input. Is it somehow possible to retrieve location field values using SharePoint Search? I cannot find a crawled property matching the name of the used location field so the question is if location fields are not indexed at all? I can get the values for the "sub fields" of the location field like City, PostalCode etc. but since there does not seem to be a "sub field" for the coordinates I am out of luck since that is what I am after.

**Comment by wobba** [comment, marked as answer]
They are not indexed afaik.
